foundations/javascript_basics/problem_solving.md

```diff
@@ -1,61 +1,68 @@
 ### Introduction
-Before we start digging into some pretty nifty JavaScript, we need to begin talking about *problem solving*: the most important skill a developer needs.
+
+Before we start digging into some pretty nifty JavaScript, we need to begin talking about _problem solving_: the most important skill a developer needs.
 
 Problem solving is the core thing software developers do. The programming languages and tools they use are secondary to this fundamental skill.
 
 V. Anton Spraul defines problem solving in programming as:
 
 > "Problem solving is writing an original program that performs a particular set of tasks and meets all stated constraints."
+
 - Think Like a Programmer
 
 The set of tasks can range from solving small coding exercises all the way up to building a social network site like Facebook or a search engine like Google. Each problem has its own set of constraints, for example, high performance and scalability may not matter too much in a coding exercise but it will be vital in apps like Google that need to service billions of search queries each day.
 
-New programmers often find problem solving the hardest skill to build. It's not uncommon for budding programmers to breeze through learning syntax and programming concepts, yet when trying to code something on their own, they find themselves staring blankly at their text editor not knowing where to start. 
+New programmers often find problem solving the hardest skill to build. It's not uncommon for budding programmers to breeze through learning syntax and programming concepts, yet when trying to code something on their own, they find themselves staring blankly at their text editor not knowing where to start.
 
 The best way to improve your problem solving ability is by building experience by making lots and lots of programs. The more practice you have the better you'll be prepared to solve real world problems.
 
 In this lesson we will walk through a few techniques that can be used to help with the problem solving process.
 
 ### Learning Outcomes
+
 By the end of this lesson, you should be able to do the following:
 
-* Explain the three steps in the problem solving process.
-* Explain what pseudo code is and be able to use it to solve problems.
-* Be able to break a problem down into subproblems.
+- Explain the three steps in the problem solving process.
```

Comment:
In this case, the asterisk is correct.

Reply:
I had prettier activated. I reverted the changes made by prettier and made a new commit.